package/gluon-xlat464-clat/Makefile

```diff
@@ -0,0 +1,38 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-xlat464-clat
+PKG_VERSION:=1
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include ../gluon.mk
+
+
+define Package/gluon-xlat464-clat
+  SECTION:=gluon
+  CATEGORY:=Gluon
```

Comment:
i'm not sure how to proceed with SECTION and category, but i just noticed some of your packages currently open for review contain this, some don't